modules/homeScreen.php

```diff
@@ -0,0 +1,10 @@
+<?php
+require_once("../classes/homescreen.php");
```

Comment:
why require once and not require

Reply:
Thanks, @basilcea. ```require_once``` is an alias for ```require```, except that ```require_once``` checks if the file has been previously included and skips if it.

Reply:
got it